src/analysis/special_functions/pow.lean

```diff
@@ -1606,7 +1606,7 @@ begin
 end
 
 @[simp] lemma one_rpow (x : ℝ) : (1 : ℝ≥0∞) ^ x = 1 :=
-by { rw [← coe_one, coe_rpow_of_ne_zero one_ne_zero], simp }
+by { rw [← @coe_one ℝ≥0, coe_rpow_of_ne_zero one_ne_zero], simp }
```

Comment:
I think this is a good argument that `coe_one` and `coe_zero` should take an explicit type argument

Reply:
I suspect you have to write manual aliases wrapping the projection.

At any rate, this can wait till a follow-up